### ISSUE TITLE
NTR: Prenatal ultrasound: hyperechogenic lungs (Lab: ultrasound & echocardiogram)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8685]
New term request Prenatal ultrasound: hyperechogenic lungs (elthat_101002181857):   UMLS: 4540049

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000252
- label: Fetal lung hyperechogenicity
- parents: [Abnormal fetal pulmonary morphology (HP:0034243)](https://hpo.jax.org/app/browse/term/HP:0034243)
- pmid: PMID:8528800
- def: Increased response (echo) of lung tissue of a fetus observed on prenatal sonography. Increased echo response is conventially displayed as a higher degree of brightness.
- comment: The ultrasonographic appearance of fetal lung hyperechogenicity is most commonly associated with congenital bronchopulmonary abnormalities, such as cystic adenomatoid malformation or pulmonary lobar sequestration.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8685
- synonyms:

**Comment by pnrobinson** [comment]
@MickeySegal -- see new term above
